### ISSUE TITLE
Error: undefined method 'localtime' for #<Date ...>

### ISSUE THREAD

**Issue by stevegeek** [ayushn21/bridgetown-sitemap#2]
When generating a sitemap it is possible for the following to occur

```
[Bridgetown] Exception raised: NoMethodError
[Bridgetown] undefined method `localtime' for #<Date: 2020-12-30 )> 
```

The reason is in `sitemap.erb`, `#localtime` is called on the return value of `Bridgetown::Resource::Base#sitemap_last_modified_at` ( `resource.sitemap_last_modified_at.localtime.xmlschema` ) but that method returns `data.last_modified_at || latest_git_commit_date || date` which can be a Date object, eg when the post front-matter includes `date: 2020-12-30`

Bridgetown v1.0.0.beta2 (codename "Pearl")

**Comment by ayushn21** [comment]
Thanks @stevegeek. Should be fixed in v1.1.2.

https://github.com/ayushn21/bridgetown-sitemap/releases/tag/v1.1.2

**Comment by stevegeek** [comment]
Thanks @ayushn21 works perfect!
